src/client/src/orangehrmRecruitmentPlugin/pages/AddJobVacancy.vue

```diff
@@ -149,6 +151,8 @@ const vacancyModel = {
   isPublished: true,
 };
 
+const basePath = `${window.location.protocol}//${window.location.host}${window.appGlobal.baseUrl}`;
```

Comment:
```suggestion
const basePath = `${window.location.protocol}://${window.location.host}${window.appGlobal.baseUrl}`;
```
:thinking:

Reply:
window.location.protocol = “http:”
protocol comes with the :

Reply:
https://css-tricks.com/snippets/javascript/get-url-and-url-parts-in-javascript/